dtfs-central-api/src/types/types-helper.ts

```diff
@@ -1 +1,5 @@
 export type ValuesOf<T> = T[keyof T];
+
+export type Prettify<T> = {
+  [K in keyof T]: T[K];
+};
```

Comment:
Do you need to add `& {}` to this type? https://www.totaltypescript.com/concepts/the-prettify-helper

Also, maybe worth adding this link (or wherever you got this from) as it's not immediately obvious how this works.